sdcm/nemesis.py

```diff
@@ -466,6 +467,27 @@ def disrupt_rolling_restart_cluster(self):
             raise UnsupportedNemesis('RollingRestartCluster is supported only for kubernetes backends')
         self.cluster.rollout_restart()
 
+    def disrupt_authenticator_switch_and_restart(self):
```

Comment:
Please add a description to function because it's not clear what you're trying to do here.

Reply:
OK.